### PR TITLE
glass: Add help button to top-bar

### PR DESCRIPTION
**Pull request by votdev**
Bildschirmfoto vom 2021-04-20 08-23-26

* Add link to website
* Add link to discussions

Related to: https://github.com/aquarist-labs/aquarist-labs.github.io/issues/14

Signed-off-by: Volker Theile <vtheile@suse.com>

## Checklist
- [ ] References issues, create if required
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins test tumbleweed`
- `jenkins run tumbleweed`
- `jenkins test leap`
- `jenkins run leap`

</details>

### CODE REVIEW

File: src/glass/src/app/core/top-bar/top-bar.component.html [hunk 1]
```diff
@@ -7,4 +7,28 @@
...
+  </button>
 </mat-toolbar>
+
+<mat-menu #helpMenu="matMenu">
+  <a href="https://aquarist-labs.github.io/"
+     target="_blank"
+     class="glass-text-no-decoration">
+    <button mat-menu-item>
+      <mat-icon svgIcon="mdi:web"></mat-icon>
+      <span translate>Homepage</span>
```
Comment (s0nea) [inline src/glass/src/app/core/top-bar/top-bar.component.html:23]: Hmmm I'm wondering if `Website` would fit better here?

Comment (jecluis) [reply src/glass/src/app/core/top-bar/top-bar.component.html:23]: Agreed. "Website" seems more appropriate.

Comment (asettle) [reply src/glass/src/app/core/top-bar/top-bar.component.html:23]: +1 to website. Otherwise, love it!

**Review by s0nea — APPROVED** [review]
:+1: thank you!

**Pull request merged**